### PR TITLE
feat: output suggested version into patch log

### LINKED ISSUE #1552: feat(installer): omit suggested version into log
This helps us quickly identify the suggested version without having to look up the compatibility list of this specific ReVanced Patches version.

### Current
```log
- Patch Info
App: com.google.android.youtube v18.45.43
Patches version: v2.202.0
Patches added: Bleep
Patches removed: Bloop
Options changed: Banana
```

### Suggested
```log
- Patch Info
App: com.google.android.youtube v18.45.43 (Suggested: vXX.XX.XX)
Patches version: v2.202.0
Patches added: Bleep
Patches removed: Bloop
Options changed: Banana
```

```log
- Patch Info
App: com.google.android.youtube
App version: vXX.XX.XX
App suggested version: vXX.XX.XX
Patches version: v2.202.0
Patches added: Bleep
Patches removed: Bloop
Options changed: Banana
```

### PR DESCRIPTION
**Pull request by validcube**
```
- Patch Info
App: com.google.android.bleep.bloop.gah v1.33.7 (Suggested: Any)
Patches version: v2.202.0
Patches added: Packground Blay, Banced MicroG support
```

Closes #1552 #1550

### CODE REVIEW

File: lib/ui/views/installer/installer_viewmodel.dart [hunk 1]
```diff
@@ -228,22 +256,22 @@ class InstallerViewModel extends BaseViewModel {
       'Model: ${info['model']}',
       'Android version: ${info['androidVersion']}',
       'Supported architectures: ${info['supportedArch'].join(", ")}',
-      'Root permissions: ${isRooted ? 'Yes' : 'No'}',
-      
+      'Root permissions: ${isRooted ? 'Yes' : 'No'}', //
```
Comment (Ushie) [inline lib/ui/views/installer/installer_viewmodel.dart:259]:
```suggestion
      'Root permissions: ${isRooted ? 'Yes' : 'No'}',
```

File: lib/ui/views/installer/installer_viewmodel.dart [hunk 2]
```diff
@@ -228,22 +256,22 @@ class InstallerViewModel extends BaseViewModel {
...
+
       '\n- Patch Info',
-      'App: ${_app.packageName} v${_app.version}',
+      'App: ${_app.packageName} v${_app.version} (Suggested: ${_getSuggestedVersion(_app.packageName)})',
       'Patches version: ${_managerAPI.patchesVersion}',
       'Patches added: ${_formatPatches(patchesAdded)}',
       'Patches removed: ${_formatPatches(patchesRemoved)}',
-      'Options changed: ${_formatPatches(patchesChanged)}',
+      'Options changed: ${_formatPatches(patchesChanged)}', //
```
Comment (Ushie) [inline lib/ui/views/installer/installer_viewmodel.dart:266]:
```suggestion
      'Options changed: ${_formatPatches(patchesChanged)}',
```

File: lib/ui/views/installer/installer_viewmodel.dart [hunk 3]
```diff
@@ -228,22 +256,22 @@ class InstallerViewModel extends BaseViewModel {
...
 
       '\n- Settings',
       'Allow changing patch selection: ${_managerAPI.isPatchesChangeEnabled()}',
       'Version compatibility check: ${_managerAPI.isVersionCompatibilityCheckEnabled()}',
       'Show universal patches: ${_managerAPI.areUniversalPatchesEnabled()}',
       'Patches source: ${_managerAPI.getPatchesRepo()}',
-      'Integration source: ${_managerAPI.getIntegrationsRepo()}',
-      
+      'Integration source: ${_managerAPI.getIntegrationsRepo()}', //
```
Comment (Ushie) [inline lib/ui/views/installer/installer_viewmodel.dart:273]:
```suggestion
      'Integration source: ${_managerAPI.getIntegrationsRepo()}',
```

**Review by Ushie — CHANGES_REQUESTED** [review]

**Comment by TheAabedKhan** [comment]
How about separating the app info into its own section? For example, use `- App Info` and place the package name, installed version, and suggested version on a new line for each.

**Comment by Ushie** [comment]
Compactness is good, it allows it to be posted in more places like Discord where there is limits on message sizes

**Review by Ushie — APPROVED** [review]

**Comment by Ushie** [comment]
> > Compactness is good, it allows it to be posted in more places like Discord where there is limits on message sizes
> 
> A downside of compactness is that in the GitHub app, code blocks don't line-wrap so you need to side-scroll. Same issue with txt files in most file readers on mobile.

You have to deal with this issue nevertheless, given that the list of patches added/removed is horizontal as well

**Pull request merged**